### ISSUE TITLE
New ESLint core rule: multiline-comment-style

### ISSUE THREAD

**Issue by mightyiam** [standard/standard#1003]
https://eslint.org/docs/rules/multiline-comment-style

Let's set a standard for multiline comments.

The default `starred-block` seems like the way to go.

**Comment by mightyiam** [comment]
Here is a PR:
https://github.com/standard/eslint-config-standard/pull/94

**Comment by uloco** [comment]
I think block comments should be only used as doc comments. In this way they can clearly be distinguished and syntax highlighted appropriately. Imoo we shouldn't enforce this rule...

**Comment by voxpelli** [comment]
@uloco Isn't there a difference between a doc comment and an ordinary comment in that one start with `/**` and the other with just plain `/*`?

https://eslint.org/docs/rules/multiline-comment-style only mentions the need to start with `/*` while eg. http://usejsdoc.org/about-getting-started.html outlines that `/**` is required for a documentation comment.

I'm +1 on this and @mightyiam can probably update his PR now that ESLint 4 (and almost ESLint 5: https://github.com/standard/eslint-config-standard/issues/123) is supported by `eslint-config-standard`

**Comment by uloco** [comment]
@voxpelli Yes you are right, there is a difference. It's just personal taste. I don't use block comments at all, because they look too familiar to doc comments and make problems when nesting (and toggling them). I use line comments for commenting code and doc comments for docs. It's just an opinion. If you really like to enforce this style, go for it.
